CHANGELOG.md

```diff
@@ -6,10 +6,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.6.3] - 2022-06-14
+
```

Comment:
I forgot to add these when i merge this PR : https://github.com/odolbeau/phone-number-bundle/pull/111
```
### Added
- Add catalan translation
```